user/deployment/s3.md

```diff
@@ -279,4 +292,4 @@ deploy:
   ..
   endpoint: https://nyc3.digitaloceanspaces.com
 ```
-{: data-file=".travis.yml"}
+{: data-file=".travis.yml"}
```

Comment:
Not sure how this line is showing up in a diff (I can't detect the white space change), but I do prefer this not being a part of the PR.

Reply:
Seems to be an oddity from fixing the conflict, but agreed.